docs/tutorial/t-deploy-mysql.md

```diff
@@ -0,0 +1,128 @@
+# Get a Charmed MySQL up and running
+
+This is part of the [Charmed MySQL Tutorial](/t/charmed-mysql-tutorial-overview/9922?channel=8.0/edge). Please refer to this page for more information and the overview of the content.
+
+## Deploy Charmed MySQL
+
+To deploy Charmed MySQL, all you need to do is run the following command, which will fetch the charm from [Charmhub](https://charmhub.io/mysql?channel=edge) and deploy it to your model:
+```shell
+juju deploy mysql --channel edge
```

Comment:
Should we?
```suggestion
juju deploy mysql --channel 8.0/edge
```

Reply:
Actually just `--channel 8.0`. I propose to submit this AS IS, to unlock future changes on Discourse.
Address all your comments on Discourse and ask Enrico to create new PR to sync Discourse->GitHub.

Reply:
Applied on discource.